src/libzippp.h

```diff
@@ -148,6 +149,8 @@ namespace libzippp {
          */
         bool open(OpenMode mode=READ_ONLY, bool checkConsistency=false);
         
+		bool open(const char* buffer, uint32_t sz, OpenMode mode = READ_ONLY, bool checkConsistency = false);
```

Comment:
- Indentation is wrong, use only spaces
- Remove spaces around `=` for the default parameters.
- Remove the useless line return